src/ipc/ipc3/handler.c

```diff
@@ -1603,6 +1603,36 @@ void ipc_boot_complete_msg(struct ipc_cmd_hdr *header, uint32_t data)
 	header->dat[1] = data;
 }
 
```

Comment:
Please use `ipc` prefix to commit subject. e.g `ipc:  Add support for SOF_IPC_FW_READY message initiated by host `

Please don't abruptly mention about jailhouse. 

I would rephrase the paragraph starting at `This flow is required because with ...`.


```This flow is required because on i.MX93, SOF runs on a A55 core and with current design
the FW is started independently and will be up before SOF driver from Linux.

We use Jailhouse to separate to isolate the cores. 
```